client/blocks/login/login-form.jsx

```diff
@@ -40,7 +40,25 @@ export class LoginForm extends Component {
 	};
 
 	componentDidMount() {
-		this.setState( { isDisabled: false } ); // eslint-disable-line react/no-did-mount-set-state
+		this.setState( { isDisabled: false }, () => { // eslint-disable-line react/no-did-mount-set-state
```

Comment:
We don't really use global state for storing ui state in Calypso, unlike in Delphin. Since it also results in less code I naturally opted for local state.